sources/gov/kaunas/municipality/educational_institutions.yaml

```diff
@@ -0,0 +1,30 @@
+id: "gov/kaunas/municipality/educational_institutions"
+title: "Educational institutions"
+description: "CSV file containing data about Kaunas educational institutions"
+type: "source"
+source:
+  - "https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv"
+  - "csv:"
+provider: "gov/kaunas/municipality"
+objects:
+  educational_institution:
+    source: https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv
```

Comment:
Here, source link should point to raw csv file: https://github.com/Kauno-miestas/Svietimas/raw/master/Svietimo%20istaigos.csv

Now it points to a html page.